advanceTask is not defined, but we say it is

mikkelrd opened the pull request:
it seems that some instructions are missing toward the end of the jQuery2 project, regarding advanceTask. we reference it on line 306 as having been defined previously in the instructions, but that doesn't seem to be the case.

as you can see, we use `advanceTask` by passing in an individual task and it does something to hopefully move it around as expected. let's start by defining the function

```
var advanceTask = function (task) {

}
```

the desired functionality can be achieved by simply making removing that task, as long as we are also adding it to another list, which appears to be covered by the instructions

```
var advanceTask = function (task) {
    $(task).remove();
}
```

that's just how i did it. i'm sure there are other ways

mikkelrd committed:
advanceTask is not defined, but we say it is

it seems that some instructions are missing toward the end of the jQuery2 project, regarding advanceTask. we reference it on line 306 as having been defined previously in the instructions, but that doesn't seem to be the case.

as you can see, we use `advanceTask` by passing in an individual task and it does something to hopefully move it around as expected. let's start by defining the function
```
var advanceTask = function (task) {

}
```

the desired functionality can be achieved by simply making removing that task, as long as we are also adding it to another list, which appears to be covered by the instructions

```
var advanceTask = function (task) {
    $(task).remove();
}
```
that's just how i did it. i'm sure there are other ways

jrobber commented:
The files are identical except some whitespace changes.  Please double check the changes you intended made it in.

mikkelrd committed:
advanceTask is not defined, but we say it is

it seems that some instructions are missing toward the end of the jQuery2 project, regarding advanceTask. we reference it on line 306 as having been defined previously in the instructions, but that doesn't seem to be the case.

mikkelrd commented:
it looks like the advanceTask function has recently been added. it seems this proposed addition is no longer necessary
